topics/query/index.md

```diff
@@ -0,0 +1,9 @@
+---
+topic: query
+display_name: Query (disambiguation)
+short_description: The term query can either mean query-string or sql-query
```

Comment:
```suggestion
short_description: The term query can either mean query-string or sql-query.
```
Fixes test failure
```

topics::query topic#test_0006_ends 'short_description' with punctuation [/home/runner/work/explore/explore/test/topics_test.rb:78]:
short_description should end with punctuation.
Expected /[.?!]\z/ to match "The term query can either mean query-string or sql-query".
```

Reply:
One would expect that I would do that properly intuitively already by now 😅